notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb

```diff
@@ -0,0 +1,602 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3jcwFPgK5BLh"
+      },
+      "source": [
+        "### Line graph"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8GvJAgnH5Nzi"
+      },
+      "source": [
+        "BigQuery DataFrames implements some of the interface required by matplotlib. This means we can pass our DataFrame right into `pyplot.plt` and using the default settings, matplotlib will draw a simple line graph for us."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "gFbCgfFC2gHw"
+      },
+      "outputs": [],
+      "source": [
+        "import matplotlib.pyplot as plt\n",
+        "\n",
+        "# matplotlin will draw a line graph by default\n",
+        "plt.plot(new_cases_usa)\n",
+        "# Rotate the labels on the x axis so that they don't overlap\n",
+        "plt.xticks(rotation=45)\n",
+        "# label the y axis for clarity\n",
+        "plt.ylabel(\"New Cases\")\n",
+        "\n",
+        "# Show the plot\n",
+        "plt.show()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "sM5-HFDx70RG"
+      },
+      "source": [
+        "## Visualization #2: Symptom-related searches compared to new cases"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "se1b6Vf4XB9_"
+      },
+      "source": [
+        "### Filter data"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Wl2o-NYMoygb"
+      },
+      "source": [
+        "We're curious if searches for symptoms like \"cough\" and \"fever\" went up in the same times and places that new COVID-19 cases occured, compared to non-symptoms like \"bruise.\" Let's plot searches vs. new cases to see if it looks like there's a correlation."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "olfnCzyg8jYi"
+      },
+      "source": [
+        "First, we select the new cases column and the search trends we're interested in."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "LqqHzjty8jk0"
+      },
+      "outputs": [],
+      "source": [
+        "symptom_data = all_data[[\"new_confirmed\", \"search_trends_cough\", \"search_trends_fever\", \"search_trends_bruise\"]]"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "b3DlJX-k9SPk"
+      },
+      "source": [
+        "Not all rows have data for all of these columns, so let's select only the rows that do."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "g4MeM8Oe9Q6X"
+      },
+      "outputs": [],
+      "source": [
+        "symptom_data = symptom_data[\n",
+        "    symptom_data[\"search_trends_cough\"].notna() &\n",
```

Comment:
this is hacky. A better approach would be df[[column0, column1]].dropna()

Reply:
True, fixed, thanks.